swagger_client/models/triage_category.py

```diff
@@ -28,7 +28,7 @@ class TriageCategory(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'tag_label': 'str',
+        'tag_label': 'TagLabel',
```

Comment:
I believe this file needs to be manually removed and the gradle task re-run to remove references to it.